xml/System.Windows.Forms/ListViewItem.xml

```diff
@@ -1646,11 +1646,17 @@ ListViewItem1.Font = New Font(ListViewItem1.Font, _
+ The value of this property depends on the value of the <xref:System.Windows.Forms.ListViewItem.ImageList%2A> property. Depending on the current value of the <xref:System.Windows.Forms.ListView.View%2A> property of the <xref:System.Windows.Forms.ListView> control associated with the item, the <xref:System.Windows.Forms.ImageList> used by the item could be one specified in the <xref:System.Windows.Forms.ListView.LargeImageList%2A> property or the <xref:System.Windows.Forms.ListView.SmallImageList%2A> property of the <xref:System.Windows.Forms.ListView> control. If the <xref:System.Windows.Forms.ListView.View%2A> property is set to <xref:System.Windows.Forms.View.LargeIcon>, the <xref:System.Windows.Forms.ImageList> specified in the <xref:System.Windows.Forms.ListView.LargeImageList%2A> property is used; otherwise, the <xref:System.Windows.Forms.ImageList> specified in the <xref:System.Windows.Forms.ListView.SmallImageList%2A> property is used. The images defined in the <xref:System.Windows.Forms.ImageList> specified in the <xref:System.Windows.Forms.ListView.SmallImageList%2A> should have the same index positions as the images in the <xref:System.Windows.Forms.ImageList> specified in the <xref:System.Windows.Forms.ListView.LargeImageList%2A> property. If the index positions are the same for both <xref:System.Windows.Forms.ImageList> controls, you can set a single index value for the <xref:System.Windows.Forms.ListViewItem.ImageIndex%2A> property and the appropriate image will be displayed regardless of the value of the <xref:System.Windows.Forms.ListView.View%2A> property of the <xref:System.Windows.Forms.ListView> control.
```

Comment:
```suggestion
 The value of this property depends on the value of the <xref:System.Windows.Forms.ListViewItem.ImageList%2A> property. Depending on the current value of the <xref:System.Windows.Forms.ListView.View%2A> property of the <xref:System.Windows.Forms.ListView> control associated with the item, the <xref:System.Windows.Forms.ImageList> used by the item could be one specified in the <xref:System.Windows.Forms.ListView.LargeImageList%2A> property or the <xref:System.Windows.Forms.ListView.SmallImageList%2A> property of the <xref:System.Windows.Forms.ListView> control. If the <xref:System.Windows.Forms.ListView.View%2A> property is set to <xref:System.Windows.Forms.View.LargeIcon>, the <xref:System.Windows.Forms.ImageList> specified in the <xref:System.Windows.Forms.ListView.LargeImageList%2A> property is used; otherwise, the <xref:System.Windows.Forms.ImageList> specified in the <xref:System.Windows.Forms.ListView.SmallImageList%2A> property is used. The images defined in the <xref:System.Windows.Forms.ImageList> specified in the <xref:System.Windows.Forms.ListView.SmallImageList%2A> property should have the same index positions as the images in the <xref:System.Windows.Forms.ImageList> specified in the <xref:System.Windows.Forms.ListView.LargeImageList%2A> property. If the index positions are the same for both <xref:System.Windows.Forms.ImageList> controls, you can set a single index value for the <xref:System.Windows.Forms.ListViewItem.ImageIndex%2A> property and the appropriate image will be displayed regardless of the value of the <xref:System.Windows.Forms.ListView.View%2A> property of the <xref:System.Windows.Forms.ListView> control.
```

Reply:
(I just added the word "property" after SmallImageList.)